downloader/downloader.go

```diff
@@ -115,11 +115,6 @@ func getVideoAudioFormats(v *youtube.Video, quality string, mimetype string) (*y
 	var videoFormat, audioFormat *youtube.Format
 	var videoFormats, audioFormats youtube.FormatList
 
-	// Default mime-type to mp4
-	if mimetype == "" {
-		mimetype = "mp4"
-	}
-
 	formats := v.Formats.Type(mimetype)
```

Comment:
```suggestion

	formats := v.Formats
	if mimetype == "" {
		formats := formats.Type(mimetype)
	}
```

Reply:
shouldnt that be `!=`? What is your problem dude?

Reply:
yes, I wanted to say that there are more changes needed around this part, I just have a quick look at your PR.